stubs/sanic.pyi

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Dict, Iterable, List, Optional, Text, Union
```

Comment:
I had to refactor the stubs and keep all of them in the same file, there were a few issues when loading the stubs